tests/Predis/Command/Redis/Search/FTCREATE_Test.php

```diff
@@ -113,6 +114,43 @@ public function testCreatesSearchIndexWithFloat16Vector(): void
         $this->assertEquals('OK', $actualResponse);
     }
 
+    /**
+     * @group connected
+     * @group relay-resp3
+     * @return void
+     * @requiresRediSearchVersion >= 2.9.0
+     */
+    public function testCreatesSearchIndexWithMissingAndEmptyFields(): void
+    {
+        $redis = $this->getClient();
+
+        $schema = [
+            new TextField(
+                'text_empty',
+                '',
+                false, false, false, '', 1, false, true
```

Comment:
Just a thought, when reading this for the first time, it is impossible to follow. A bunch of random `true` and `false` values... Does PHP have named parameters?

Reply:
PHP have it since 8.0, we provide a support from 7.4